src/brasil/gov/temas/themes/amarelo/index.html

```diff
@@ -166,7 +166,7 @@
 
             <!-- Column 1 -->
             <div id="navigation">
-                <a name="anavigation" id="anavigation"></a>
+                <a name="anavigation" id="anavigation" class="anchor">menu</a>
```

Comment:
the use of the `name` attribute is deprecated in HTML5:

https://www.w3.org/TR/html5/obsolete.html#obsolete

Reply:
We didn't add this attribute, it was already available in the code. Since removing this is out of the scope of this PR, we're opening a new ticket do address it https://github.com/plonegovbr/brasil.gov.temas/issues/141 in another branch.